Add new activity for releasing the standard (#1159)
publiccodenet/about: activities/releasing-the-standard/index.md

---
type: Index
---

# Releasing the Standard for Pubic Code

## Preparing a release

Start by creating an issue in the Standard for Public Code repository.

> The issue can include a target release date that we're aiming for and the new version number.

Reviewer: Is this a 'can' (optional?) or a 'must'?

```suggestion
The issue must include the new version number and can include a target release date.
```

Author: It's an optional. It might also change while reviewing all the changes.

> Also include a mention of the motivation for the release, to give context.

Reviewer: ```suggestion
Also include the reason for the release, to give context.
```

> As we're still below version 1.0, we have the liberty to release early and often.

Reviewer: ```suggestion
As we're still below version 1.0, we're free to release early and often.
```

## Creating the release

Follow the [instructions](https://github.com/publiccodenet/standard/blob/develop/docs/releasing.md) in the documentation.

> ## Follow-up of a release

Reviewer: ```suggestion
## After the release
```
